### PR TITLE
Fix issue 19507 - Frontend Minicart dropdown alignment issue

### PR DESCRIPTION
**Pull request by SyntaxKay**
### Description (*)

Frontend Minicart dropdown alignment issue
### Fixed Issues (if relevant)

1.magento/magento2#19507: Frontend Minicart dropdown alignment issue

### Manual testing scenarios (*)

1.Check Minicart dropdown alignment

### Contribution checklist (*)
 - [x] Pull request has a meaningful description of its purpose
 - [x] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds on Travis CI are green)

### CODE REVIEW

**Comment by magento-engcom-team** [comment]
Hi @speedy008. Thank you for your contribution 
Here is some useful tips how you can test your changes using Magento test environment.
Add the comment under your pull request to deploy test or vanilla Magento instance:
- `@magento-engcom-team give me test instance` - deploy test instance based on PR changes
- `@magento-engcom-team give me 2.3-develop instance` - deploy vanilla Magento instance

For more details, please, review the [Magento Contributor Assistant](https://devdocs.magento.com/guides/v2.2/contributor-guide/contributing.html#contributor-assist) documentation

**Comment by Karlasa** [comment]
@magento-engcom-team give me test instance

**Comment by magento-engcom-team** [comment]
Hi @Karlasa. Thank you for your request. I'm working on Magento instance for you

**Comment by magento-engcom-team** [comment]
Hi @Karlasa, here is your [new Magento instance](https://pr-19508.instances.magento-community.engineering).
Admin access: [https://pr-19508.instances.magento-community.engineering/admin](https://pr-19508.instances.magento-community.engineering/admin)
Login: `admin` Password: `123123q`

File: app/design/frontend/Magento/luma/Magento_Checkout/web/css/source/module/_minicart.less [hunk 1]
```diff
@@ -417,6 +422,12 @@
         .block-minicart {
             right: -15px;
             width: 390px;
+            &:before {
```
Comment (Karlasa) [inline app/design/frontend/Magento/luma/Magento_Checkout/web/css/source/module/_minicart.less]: Please follow coding standards: https://devdocs.magento.com/guides/v2.3/coding-standards/code-standard-less.html
Add new line before &:before and &:after

File: app/design/frontend/Magento/luma/Magento_Checkout/web/css/source/module/_minicart.less [hunk 2]
```diff
@@ -129,7 +129,12 @@
 
         .block-minicart {
             .lib-css(padding, 25px @minicart__padding-horizontal);
-            
+            &:before {
```
Comment (Karlasa) [inline app/design/frontend/Magento/luma/Magento_Checkout/web/css/source/module/_minicart.less]: can be changed in line 115 as @_dropdown-list-pointer-position-left-right instead

Comment (Karlasa) [reply app/design/frontend/Magento/luma/Magento_Checkout/web/css/source/module/_minicart.less]: also 10/11px looks bit off in mobile without products maybe try around 4-6px instead?

**Review by Karlasa — CHANGES_REQUESTED** [review]

**Comment by SyntaxKay** [comment]
Hi @Karlasa 
I have updated CSS code Please review it.

File: app/design/frontend/Magento/luma/Magento_Checkout/web/css/source/module/_minicart.less [hunk 3]
```diff
@@ -129,7 +129,6 @@
 
         .block-minicart {
             .lib-css(padding, 25px @minicart__padding-horizontal);
-
```
Comment (Karlasa) [inline app/design/frontend/Magento/luma/Magento_Checkout/web/css/source/module/_minicart.less]: add newline back please :)

File: app/design/frontend/Magento/luma/Magento_Checkout/web/css/source/module/_minicart.less [hunk 4]
```diff
@@ -415,7 +414,7 @@
         margin-left: 13px;
 
         .block-minicart {
-            right: -15px;
+            right: -10px;
```
Comment (Karlasa) [inline app/design/frontend/Magento/luma/Magento_Checkout/web/css/source/module/_minicart.less]: you can remove this line as in line 113 it's set to -10px already

Comment (SyntaxKay) [reply app/design/frontend/Magento/luma/Magento_Checkout/web/css/source/module/_minicart.less]: @Karlasa 
I have remove Duplicate code.

**Review by Karlasa — APPROVED** [review]
Sorry for nitpicking, you did great job. Well done :)

**Review by VladimirZaets — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @VladimirZaets, thank you for the review.
[ENGCOM-3721](https://jira.corp.magento.com/browse/ENGCOM-3721) has been created to process this Pull Request

**Comment by magento-engcom-team** [comment]
Hi @speedy008. Thank you for your contribution. 
We will aim to release these changes as part of 2.3.1. 
Please check the release notes for final confirmation.

**Pull request merged**